Added SpeedTunneller

crafter6789 opened the pull request:
A faster tunneller, unable to bypass NoCheat+
This is somewhat a copy and paste from other nuker-like modules, but is easier to manage that FlatSpeedNuker. Probably not too clean.

crafter6789 committed:
Added SpeedTunneller

A faster tunneller, unable to bypass NoCheat+

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Wurst-Imperium/Wurst-MC-1.11?pullRequest=1) <br/>All committers have signed the CLA.

crafter6789 committed:
Cleaned up SpeedTunneller

crafter6789 committed:
Added  several utility mods

Bounded Nuker
More Block Reach
Spectator Detector

**.bound command is unstable**
**Spectator Detector is untested**

crafter6789 committed:
Merge remote-tracking branch 'refs/remotes/Wurst-Imperium/master' into bounded-nuker

crafter6789 committed:
Revert "Added  several utility mods"

This reverts commit bf2d3082d370e885eb49382133b8158b9ace0993.

crafter6789 committed:
Revert "Revert "Added  several utility mods""

This reverts commit d3f49aefc4537ac17b1a2d64845e8b218eca5c20.

crafter6789 committed:
Fixed accidantal commit

My ineptitude

crafter6789 committed:
Several mods

BowSpam

crafter6789 committed:
Bounded nuker (#1)

* Remove cost field from PathPoint

* getDistance() > getHeuristic()

* Move previous positions to map

* :gem: Simplify queue comparator

* Add PathPos

* :gem: Clean up PathFinder queue

* :gem: Clean up PathFinder

* :gem: Clean up fall damage check

* Update PathFinder

* Update PathFinder

* Move GoTo functionality to PathFinder

* :gem: Remove unnecessary "GL11."

* Add path validation

* Change version to 5.1

* :bug: Fix .goto not stopping when updating the path

* Added  several utility mods

Bounded Nuker
More Block Reach
Spectator Detector

**.bound command is unstable**
**Spectator Detector is untested**

* Revert "Added  several utility mods"

This reverts commit bf2d3082d370e885eb49382133b8158b9ace0993.

* Revert "Revert "Added  several utility mods""

This reverts commit d3f49aefc4537ac17b1a2d64845e8b218eca5c20.

* Fixed accidantal commit

My ineptitude

* Several mods

BowSpam

crafter6789 committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into bounded-nuker

crafter6789 committed:
Merge pull request #2 from crafter6789/bounded-nuker

Bounded nuker

DesWurstes commented:
Please squash.
